### ISSUE TITLE
Latest crate versioni outdated

### ISSUE THREAD

**Issue by torokati44** [serayuzgur/crates#201]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Additional context**

**Comment by AurevoirXavier** [comment]
I encountered the same bug. It looks like the reason is enabling `spase`?

**Comment by djc** [comment]
Yes, I have the impression the sparse index isn't supported yet?

**Comment by serayuzgur** [comment]
Crates version ?

**Comment by AurevoirXavier** [comment]
> Crates version ?

v0.6.0 from vscode.

**Comment by torokati44** [comment]
Same here.

**Comment by torokati44** [comment]
Fix confirmed, thank you for the quick response!
